packages/flutter_tools/lib/src/reporting/usage.dart

```diff
@@ -467,3 +465,154 @@ class LogToFileAnalytics extends AnalyticsMock {
     _sessionValues[param] = value.toString();
   }
 }
+
+class TestUsage implements Usage {
```

Comment:
Can we move this class to `test/lib/src` now that it's not a factory?  I don't like that we have so much test code cluttering up the real objects we ship to prod.

Reply:
Good call, will do

Reply:
done